error.vue

```diff
@@ -6,29 +6,20 @@ const { error } = defineProps<{
   error: Partial<NuxtError>
 }>()
 
+const { t } = useI18n()
 // add more custom status codes messages here
 const errorCodes: Record<number, string> = {
-  404: 'Page not found',
+  404: t('Sorry, we can\'t find that page'),
+  500: t('Sorry, something went wrong'),
```

Comment:
@wtfluckey I am not familiar enough with this repo so please forgive me if I am asking this erroneously.  It seems like we need to define these keys directly in the [en.json](https://github.com/MozillaSocial/elk/blob/main/locales/en.json) and then use the keys in the `t(<key>)` ... and I interpreting that incorrectly?  Is there a parsing script that does that for us here?

Reply:
@collectedmind It looks like this is a manual process of adding the keys to `en.json` right now. If there is a script that will do it automatically, I wasn't able to find anything in any of the readmes, the elk contributing guide or the vue or nuxt docs, and it looks like both Joseph and Anthony have been adding their strings manually as well.

So I added them! Great catch on this and apologies for missing it!

Reply:
I've just been doing find+replace work 😅